WireAPI/Sources/WireAPI/Models/FeatureConfig/EndToEndIdentityFeatureConfig.swift

```diff
@@ -34,4 +34,12 @@ public struct EndToEndIdentityFeatureConfig: Equatable, Codable {
 
     public let verificationExpiration: UInt
 
+    /// The crl proxy URL
+
+    public let crlProxy: String?
+
+    /// Uses proxy on mobile
+
+    public let useProxyOnMobile: Bool?
```

Comment:
Could these also be non optional with a default value?

Reply:
Looking at Android, we can make `useProxyOnMobile` non optional and keep `crlProxy` optional though

Reply:
fixed